CMakeLists.txt

```diff
@@ -618,7 +630,23 @@ enable_clang_tidy(
         -readability-container-data-pointer
         -readability-identifier-length
         -readability-suspicious-call-argument
-
+        #TODO Code Quality WORKAROUND ROCm 5.3 && Ubuntu 22.04 && C++17 && cppcheck 2.9 update
```

Comment:
@JehandadKhan  @atamazov these lines and the section immediately below.
I think we need to distribute the work and eliminate these warnings from code. Suppressing them for now and I think a few of them need to be suppressed anyway.